doc/Language/classtut.pod6

```diff
@@ -300,7 +287,7 @@ without having to write the method by hand. You are free instead to write
 your own accessor method, if at some future point you need to do something
 more complex than returning the value.
 
-X<|traits,is rw>
+X<|Tutorial,is rw>
```

Comment:
I'm not sure about that.

`is rw` as a trait is indexed in another place, in fact in three different places:

```
doc/Type/Attribute.pod6
104:=head2 X<trait is rw|Traits,is rw (Attribute)>

doc/Language/typesystem.pod6
424:=head3 trait X«C<is rw>|Traits,is rw (class)»

doc/Type/Signature.pod6
1084:X<|Traits,is rw>
```

So we have explanation of `is rw trait` for attributes, `is rw trait` for classes and `is rw trait` when used in signatures.

And this `is rw` is on the tutorial page, so it makes sense to put it under the tutorial term, as it's a quick introduction.

If it would be `Traits,is rw`, it would conflict with the anchor elsewhere, and when you have a search giving ambiguous results it's not that awesome. Am I wrong?

Reply:
Well, it was originally traits, is rw. You would be simply capitalizing it. My point is that you're substituting some terms by more general ones, thus losing search information.